docs/ug/customizingReports.md

```diff
@@ -59,3 +59,17 @@ In both instances, it is **necessary to commit any changes** for them to be dete
 </box>
 
 3\. Add a git `.mailmap` file at the top-level of the repository, specifying mapped authors/commiters and/or e-mail addresses as per [gitmailmap documentation](https://git-scm.com/docs/gitmailmap). Any mappings specified here will be applied by git before all other RepoSense configurations. Configuration via `.mailmap` is particularly useful if you want the mapping to apply for all git commands as well instead of just for RepoSense.
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+### Add a Title
+
+A title component can be added by creating a file titled `title.md` in the assets directory. You can specify the assets directory according to the reference below:
+  {{ embed("Appendix: **CLI syntax reference**", "cli.md") }}
+
+The title can render a combination of HTML and plaintext ([example](title.md)), and will appear on the top of the left panel as shown below:
```

Comment:
The example title file here opens up within the user guide documentation itself as another page. Instead, it should appear as code, perhaps pointing to the code file in GitHub. Can you change the link here accordingly?